app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -66,17 +66,17 @@ public function cleanupQuotes() {
      */
     public function cleanupOrders() {
         try {
-            $expiration_time = Mage::getSingleton('core/date')
-                ->gmtDate('Y-m-d H:i:s', time()-(60*PRE_AUTH_STATE_TIME_LIMIT_MINUTES));  // Magento uses GMT to save in DB
+            $expiration_time = gmdate('Y-m-d H:i:s', time()-(60*self::PRE_AUTH_STATE_TIME_LIMIT_MINUTES));  // Magento uses GMT to save in DB
```

Comment:
Just wondering, why did you get rid of the getSIngleton?

Reply:
@adentong 
The mode was doing some locale based conversion on the dates that was not necessary as the time() function is returning things in GMT already.  Using the model function is an error in this context.

https://github.com/OpenMage/magento-mirror/blob/magento-1.9/app/code/core/Mage/Core/Model/Date.php#L137-L164

https://www.php.net/manual/en/function.time.php

Reply:
Interesting!